src/app/components/home/home.component.html

```diff
@@ -2,7 +2,7 @@
   <i class="fa fa-bars bar-toggle" (click)="drawer.toggle()"></i>
   <div>
     <img class="logo" src="../../../assets/home/VV_Logo_Reversed@2x.png" alt="Voter's Voice Initiative" width="120" height="auto"
-      routerLinkActive="active-link" routerLink="/splash">
+      routerLinkActive="active-link" routerLink="/splash" (click)="setOfficeStep(-1)">
```

Comment:
I do not think this click is necessary here.  SetofficeStep function is to track which candidate or subcandidate is chosen.